### PR TITLE
doc: add the damage types that scrub can repair

### PR DESCRIPTION
**Pull request by neesingh-rh**
Fixes: https://tracker.ceph.com/issues/51459
Signed-off-by: Neeraj Pratap Singh <neesingh@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by zdover23 — APPROVED** [review]

File: doc/cephfs/scrub.rst [hunk 1]
```diff
@@ -131,3 +131,15 @@ Control (ongoing) File System Scrubs
...
+Damages
+=======
+
+The damages that can be reported and repaired by File System Scrub are:
+
+* DENTRY
+
+* DIR_FRAG
+
+* BACKTRACE
```
Comment (rishabh-d-dave) [inline doc/cephfs/scrub.rst:144]: Should we add more details about each type of damage?

Comment (neesingh-rh) [reply doc/cephfs/scrub.rst]: PTAL

Comment (zdover23) [reply doc/cephfs/scrub.rst]: LGTM. Can this be backported to Quincy and to Pacific?

**Review by rishabh-d-dave — APPROVED** [review]
Looks much better now. Thanks! If possible, please add more details.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/49932 - Quincy backport
https://github.com/ceph/ceph/pull/49933 - Pacific backport

**Pull request merged**